apps/test/unit/makerlab/TouchSensorTest.js

```diff
@@ -0,0 +1,105 @@
+/** @file Test the TouchSensor maker component which wraps Touchpad */
+import {expect} from '../../util/configuredChai';
+import sinon from 'sinon';
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+import TouchSensor from '@cdo/apps/makerlab/TouchSensor';
+
+const DOWN = 'down';
+const HOLD = 'hold';
+const UP = 'up';
+
+describe('TouchSensor', function () {
+  let fakeTouchPad, touchSensor, spy;
```

Comment:
not sure whether/if we call this out in our style doc, but i think we usually put each of these on separate lines.